assets/js/modules/adsense/datastore/tags.js

```diff
@@ -73,20 +57,101 @@ const fetchGetTagPermissionStore = createFetchStore( {
 	},
 } );
 
+// Actions
+const GET_EXISTING_TAG = 'FETCH_EXISTING_TAG';
+const RECEIVE_EXISTING_TAG = 'RECEIVE_EXISTING_TAG';
+const WAIT_FOR_EXISTING_TAG = 'WAIT_FOR_EXISTING_TAG';
+
 const BASE_INITIAL_STATE = {
 	existingTag: undefined,
 	tagPermissions: {},
 };
 
+const baseActions = {
+	getExistingTag() {
+		return {
+			payload: {},
+			type: GET_EXISTING_TAG,
+		};
+	},
+	receiveGetExistingTag( existingTag ) {
+		invariant(
+			existingTag === null || isValidClientID( existingTag ),
+			'existingTag must be a valid property ID or null.'
+		);
+
+		return {
+			payload: { existingTag },
+			type: RECEIVE_EXISTING_TAG,
+		};
+	},
+	waitForExistingTag() {
+		return {
+			payload: {},
+			type: WAIT_FOR_EXISTING_TAG,
+		};
+	},
```

Comment:
Let's make sure any actions with controls are always generators that yield 👍  (see `getExistingTag` above too)

Reply:
@aaemnnosttv I updated most of these, but `fetchGetExistingTag` breaks if it's made into a generator. I think it's because we're yielding it inside the resolver.